packages/@aws-containers/aws-ecs-builder/lib/extensions/extension-interfaces.ts

```diff
@@ -0,0 +1,218 @@
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as cdk from '@aws-cdk/core';
+import { Service } from '../service';
+
+/**
+ * A list of the capacity types that are supported. These
+ * capacity types may change the behavior of an extension.
+ */
+export enum EnvironmentCapacityType {
```

Comment:
In general CDK tends to have wordy enum's, which do have the minor effect of also being slightly easier to search for because they are unique. I don't have strong opinions on this though, so willing to change if you feel strongly about it.